blog/docs/releases/announcing-knative-v1-6-release.md

```diff
@@ -0,0 +1,156 @@
+---
+title: "v1.6 release"
+linkTitle: "v1.6 release"
+Author: "Evan Anderson and Paul Schweigert"
+Author handle: https://github.com/dprotaso
+date: 2022-07-12
+description: "Knative 1.6 release announcement"
+type: "blog"
+---
+
+## Announcing Knative 1.6 Release
+
+A new version of Knative is now available across multiple components.
+
+Follow the instructions in [Installing Knative](https://knative.dev/docs/install/) to install the components you require.
+
+This release brings a number of smaller improvements to the core Knative Serving and eventing components, and several improvements to specific plugins, in particular `net-kourier`, `net-gateway-api`, and `eventing-rabbitmq`.
```

Comment:
```suggestion
This release brings a number of smaller improvements to the core Knative Serving and Knative Eventing components, and several improvements to specific plugins, in particular `net-kourier`, `net-gateway-api`, and `eventing-rabbitmq`.
```

Reply:
You can remove "Knative" here if you want but let's keep the capitalization and format consistent at least

Reply:
Done.